next.clayui.com/content/docs/css/components/list.md

```diff
@@ -0,0 +1,388 @@
+---
+title: "List"
+description: "Lists are a visual representation of a dataset, based on groups of related content, that is organized vertically."
+---
+
+<div class="nav-toc">
+
+- [Loading Icon Dotted](#loading-icon-dotted)
+- [Loading Icon Dotted Light](#loading-icon-dotted-light)
+- [Button with Loading Icon Dotted](#button-with-loading-icon-dotted)
+
+</div>
+
+<div class="clay-site-alert alert alert-info">
+	Check the <a href="https://liferay.design/lexicon">Lexicon</a> <a href="https://liferay.design/lexicon/core-components/list/">List Pattern</a> for a more in-depth look at the motivations and proper usage of this component.
+</div>
+
+### List Groups
+
+#### Basic Example 
+
+The most basic list group is an unordered list with list items and the proper classes. Build upon it with the options that follow, or with your own CSS as needed.
+
+<div class="sheet-example">
+    <ul class="list-group">
+        <li class="list-group-item">Cras justo odio</li>
+        <li class="list-group-item">Dapibus ac facilisis in</li>
+        <li class="list-group-item">Morbi leo risus</li>
+        <li class="list-group-item">Porta ac consectetur ac</li>
+        <li class="list-group-item">Vestibulum at eros</li>
+    </ul>
+</div>
+
+```html
+<ul class="list-group">
+    <li class="list-group-item">Cras justo odio</li>
+    <li class="list-group-item">Dapibus ac facilisis in</li>
+    <li class="list-group-item">Morbi leo risus</li>
+    <li class="list-group-item">Porta ac consectetur ac</li>
+    <li class="list-group-item">Vestibulum at eros</li>
+</ul>
+```
+
+#### List Group Item Flex
+
+Align content inside `.list-group-item` with flexbox with `.list-group-item-flex`.
+
+Use a combination of `.autofit-col` and `.autofit-col.autofit-col-expand` to control the size of each cell.
+
+> `show-quick-actions-on-hover` will need Javascript to add class `table-focus` on the table row when an item inside is focused to make it keyboard accessible.
+<div class="sheet-example">
```

Comment:
Should there be a newline above this? Or is this just weird GH coloring?

Reply:
Shouldn't. I sent this file incorrectly

Reply:
I worked on this yesterday and sending in few minutes ⌛️